prototype scripts/llm_judge.py

```diff
@@ -0,0 +1,56 @@
+# %%
+import cohere 
+import re
+import pandas as pd
+
+# %%
+co = cohere.Client(
+  api_key="", # This is your API key
+) 
+
+# Define rate limit: 10 calls per 60 seconds
+# CALLS = 5
+# PERIOD = 60
+
+# @sleep_and_retry
+# @limits(calls=CALLS, period=PERIOD)
+def eval(row):
+  inputprompt = row["inputs"]
+  answer = row["targets"]
+
+  print(inputprompt, answer)
+
+  response = co.chat( 
+    model='c4ai-aya-23-8b',
+    message=f'''[System]
+Please act as an impartial judge and evaluate the quality of the response provided by an
+AI assistant to the user question displayed below. Your evaluation should consider factors
```

Comment:
Let's experiment with using the preamble parameter as well: https://docs.cohere.com/docs/preambles

Reply:
I tried that when testing on the dashboard. It didn't even follow the preamble.

Reply:
Also doing in-context in the message doesn't make the model perform any worse, so I don't think this needs to change.